packages/@react-spectrum/tag/stories/TagGroup.stories.tsx

```diff
@@ -54,7 +56,52 @@ export default {
         disable: true
       }
     },
-    maxRows: {type: 'number'}
+    onAction: {
+      table: {
+        disable: true
+      }
+    },
+    items: {
+      table: {
+        disable: true
+      }
+    },
+    maxRows: {
+      type: 'number'
+    },
+    contextualHelp: {
+      table: {
+        disable: true
+      }
+    },
+    isRequired: {
+      control: 'boolean'
+    },
+    necessityIndicator: {
+      control: 'select',
```

Comment:
Why is this control a `select`? In other stories the control for this prop is a radio. We should pick one and use everywhere. Ditto for other controls here.

Reply:
Weighing in here, I like the select more since it takes less room, especially if there are a large number of controls.